from errors.base import WorkflowsBaseException

class ContextError(WorkflowsBaseException):
    pass
